docs/conf.py

```diff
@@ -110,7 +111,7 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'alabaster'
+html_theme = 'sphinx_rtd_theme'
```

Comment:
Let's go with [Furo](https://pypi.org/project/furo/) for now, I like it better as a default.